app/liquid_tags/vimeo_tag.rb

```diff
@@ -27,7 +27,6 @@ def render(_context)
 
   def get_id(input)
     match = input.match(REGISTRY_REGEXP)
-    # binding.pry
```

Comment:
Oops! left this in a previous PR, taking it out now 🤦🏾‍♀️

Reply:
I think there's a rubocop check to make sure you don't inadvertently leave it uncommented, good catch.